src/content/whats-new/2023/05/whats-new-05-05-dive-into-nrql-from-logs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Unleash your log data analysis with NRQL'
```

Comment:
This should be more plainly descriptive than flowery, IMO. How about "Advanced logs analysis using NRQL, one click away"?

Reply:
👍